gatsby_fp/src/components/Layout.js

```diff
@@ -103,6 +103,7 @@ export const facultyData = graphql`
         pagenum
         status
         title
+        title_secondary
```

Comment:
This has nothing to do with making the logo a link. Something got messed up with this PR